espnet2/asr_transducer/encoder/modules/positional_encoding.py

```diff
@@ -0,0 +1,91 @@
+"""Positional encoding modules."""
```

Comment:
This class can also be moved to a general directory since ASR Conformer also uses relative positional encoding.
Besides, I am not sure we should make ESPnet2 more close-loop or use definitions in `espnet/nets/pytorch_backend/transformer/embedding.py`.

Reply:
> This class can also be moved to a general directory since ASR Conformer also uses relative positional encoding.

This one is mostly equivalent to `StreamPositionalEncoding(...)`, yes!
If it's okay I would like to keep it despite the duplication as one of my next PR will further modify the class. If you think there are no major revision to warrant a duplication, I'll extend the class from `embedding.py`!

> Besides, I am not sure whether to make ESPnet2 more close-loop or use definitions in espnet/nets/pytorch_backend/transformer/embedding.py.

This is only my opinion but I would rather have duplicated code in different versions than depending on classes from previous version. It can bring a lot of dependencies (in terms of design, convention, etc) and it become difficult in terms of maintenance. Also, the code is sometimes overloaded due to adding support for new thing over time or because some things were implemented without a "global" purpose at time.

Reply:
It is okay for me to keep the current setups :)